spec/controllers/trn_submissions_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-describe TrnSubmissionsController do
+xdescribe TrnSubmissionsController do
```

Comment:
Any reason why we're ignoring this?

Reply:
Yeah, it's currently creating a mocked version of a trainee which has met some theoretical criteria for submission - which means we can't authenticate against it, as it's not really a trainee object. Once we've got the submission process using the trainee factory, the authentication should be seamless, but it felt like a lot of work to rewrite this test, which doesn't really test a working function

Reply:
Changed now to include this test, with a bit of refactoring